src/Core/Framework/DataAbstractionLayer/FieldSerializer/PriceDefinitionFieldSerializer.php

```diff
@@ -202,6 +202,12 @@ private function validateRules(array $data, string $basePath): ConstraintViolati
             if (!$rule instanceof Container) {
                 $rule->assign($data);
                 $validations = $rule->getConstraints();
+                $config = $rule->getVars() ?? [];
+                if(is_array($config) && array_key_exists('operator', $config) && array_key_exists('amount', $config)) {
```

Comment:
Can you run `composer run ecs-fix` to fix the code-style here?

Reply:
No problem :)